pre-registration/pre-registration-application-service/src/test/resources/application.properties

```diff
@@ -84,7 +84,7 @@ auth.server.validate.url=
 global.config.file=application-mz.properties
 pre.reg.config.file=pre-registration-mz.properties
 
-# spring.cloud.config.uri=https://dev.mosip.net/config
+spring.cloud.config.uri=https://dev.mosip.net/config
```

Comment:
If this is not given, is it trying to connect to localhost and failing ?

Reply:
Yes, it's trying to fetch application.properties and pre-registration-mz.properties either from local host config or dev config in pre-registration all services they are not using localhost config so i used dev config in localhost:51000/config also it's working.